### ISSUE TITLE
ES module support

### ISSUE THREAD

**Issue by nunoarruda** [mariusGundersen/sparkline#9]
Hi @mariusGundersen ,

Thanks for the handy library, any chance you can make it importable as an ECMAScript module? Angular, for example, complains about CommonJS or AMD dependencies:

```
[1] WARNING in /Users/nunoarruda/repos/Pulse-Angular/src/app/devices/device-card-active/device-card-active.component.ts depends on 'sparklines'. CommonJS or AMD dependencies can cause optimization bailouts.
[1] For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
```

Here's why: https://angular.io/guide/build#configuring-commonjs-dependencies

**Comment by mariusGundersen** [comment]
Hmm, yes that should not be too hard. I'm not very familiar with angular, but it should be possible to have a module entry in package.json that points to the es-module version of the code, and then angular build tool uses that file instead. That way the change won't break anything

**Comment by mariusGundersen** [comment]
I've published v1.3.0, check if it works better now

**Comment by nunoarruda** [comment]
Thanks @mariusGundersen no more warning now 👍 

For everyone else looking at this issue, in my case, I had to:
1. Update the dependency to v1.3.0
2. Change `import * as Sparklines from 'sparklines';` to `import Sparkline from 'sparklines';`
3. Rename any `new Sparklines()` call to `new Sparkline()`
